data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java

```diff
@@ -47,7 +47,7 @@ public void singleEuropePMCLinkFromMockGraphDB() {
 	public void correctEuropePMCLinkXMLElement() throws ParserConfigurationException {
 		final String PATHWAY_DISPLAY_NAME = "p53-Dependent G1 DNA Damage Response";
 		final String PATHWAY_STABLE_ID = "R-HSA-69563";
-		final String PATHWAY_URL = "https://www.reactome.org/PathwayBrowser/#/" + PATHWAY_STABLE_ID;
+		final String PATHWAY_URL = "https://reactome.org/content/detail/" + PATHWAY_STABLE_ID;
```

Comment:
Could this become `ReactomeURLConstants.CONTENT_DETAIL_URL + PATHWAY_STABLE_ID`?

Reply:
Yeah, I think it can.  I guess I'm just used to literals in tests!